http-svr/http.ml

```diff
@@ -93,22 +91,25 @@ let output_http fd headers =
 	Unixext.really_write_string fd
 		(String.concat "" (List.map (fun x -> x ^ "\r\n") headers))
 
+let explode str = Astring.String.fold_right (fun c acc -> c :: acc) str []
+let implode chr_list = String.concat "" (List.map Astring.String.of_char chr_list)
+
 let urldecode url =
-    let chars = String.explode url in
```

Comment:
Should we use the `Uri` module here? Probably as a separate PR though, and after running lots of tests..

Reply:
I didn't know about it. Sounds like a good improvement, I'll make a ticket for it